src/Symfony/Component/Notifier/Bridge/RocketChat/composer.json

```diff
@@ -23,7 +23,7 @@
     "autoload": {
         "psr-4": { "Symfony\\Component\\Notifier\\Bridge\\RocketChat\\": "" },
         "exclude-from-classmap": [
-            "/Tests/"
+            "/Notifier/Bridge/Mattermost/Tests/"
```

Comment:
?

Reply:
We can yes, I would recommend to do this across all bridges.

But what is the difference?

Reply:
Sorry this looks wrong :-)

Reply:
Fixed